apps/client/app/components/navbars/SideNavbar.tsx

```diff
@@ -0,0 +1,101 @@
+
+import { NavLink } from "@remix-run/react";
+import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/solid";
+import { useState } from "react";
+import { NavbarLinks } from "../../../models/navbarLinks.model";
+
+interface Props {
+  links?: NavbarLinks[]
+}
+
+const defaultLinks: NavbarLinks[] = [
+  {
+    path: "/emails",
+    label: "Emails"
+  },
+  {
+    path: "/templates",
+    label: "Templates"
+  },
+  {
+    path: "/groups",
+    label: "Groups"
+  },
+  {
+    path: "/api-keys",
+    label: "API Keys"
+  },
+  {
+    path: "/docs",
+    label: "Docs"
+  },
+  {
+    path: "/help",
+    label: "Help"
+  },
+  {
+    path: "/settings",
+    label: "Settings"
+  },
+  {
+    path: "/user",
+    label: "User name"
+  },
+]
```

Comment:
maybe move this out of the component to the navbarLinks model

Reply:
for `/constants` could be better